state.go

```diff
@@ -100,7 +100,7 @@ func (s *State) NewNode(label string, info []byte) (*Node, error) {
 	}
 	_, err = s.db.Exec(
 		`INSERT INTO nodes(label, obm_info)
-			VALUES (?, ?)`,
+			VALUES ($1, $2)`,
```

Comment:
why was this just `?` earlier?

Reply:
I think postgres doesn't support the `?` notation. Not 100% sure on that, but I believe that was the motivation for the change anyway.

Reply:
Yeah, postgres was complaining about the `?` notation.